ui/source/js/components/piece.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 
 function Piece(props) {
+
+	function getVisibleClass() {
```

Comment:
The `classnames` npm package can help reduce the amount of code needed to generate conditional CSS class names.